jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -24,7 +24,6 @@ def buildExclusions = [
   [ /debian8-x86/,                    anyType,     gte(10) ], // 32-bit linux for <10 only
   [ /debian8/,                        anyType,     gte(13) ],
   [ /^ubuntu1804/,                    anyType,     lt(10)  ], // probably temporary
-  [ /^ubuntu1204/,                    anyType,     gte(10) ],
```

Comment:
We need to make sure that all jobs referencing these nodes are updated before committing this otherwise they will start scheduling jobs against them (since this script is "don't allow if" so if the line is removed it is allowed).